downstream/modules/platform/con-ocp-supported-install.adoc

```diff
@@ -32,6 +32,15 @@ If existing components have already been deployed, you must specify these compon
 * Same with {ControllerName} and {EDAName}
 |===
 
+== Modifying the number of simultaneous rulebook activations during or after {EDAcontroller} installation
+
+* If you plan to install {EDAcontroller} on {OCPShort} and modify the number of simultaneous rulebook activations, add the required `EDA_MAX_RUNNING_ACTIVATIONS` parameter. By default, {EDAcontroller} allows 12 activations per node to run simultaneously. See the example in appendix link:{URLOperatorInstallation}#eda_max_running_activations[EDA_MAX_RUNNING_ACTIVATIONS]. 
+
```

Comment:
@Alex-Izquierdo - Please review the updated content here for OCP. I updated this one-step procedure based on feedback you provided in a different module yesterday.